hw/ip/usbdev/data/usbdev.hjson

```diff
@@ -265,7 +265,7 @@
     { name: "link_out_err",
       desc: '''
             Raised if a packet to an OUT endpoint started to be received but was then dropped due to an error.
-            This error is raised if either the data toggle, token, packet or CRC is invalid or if there is no buffer available in the Received Buffer FIFO.
+            This error is raised if the data toggle, token, packet and/or CRC are invalid, if the Available Buffer FIFO is empty or if the Received Buffer FIFO is full.
```

Comment:
I agree, it's not an error and the interrupt is overloaded. FIFO overflow/underrun shouldn't happen in practice, and it is a condition of which software/developers should be aware because then the software may be altered to avoid it. I just felt it worth improving the description to clarify this.